ee/ucp/admin/configure/run-only-the-images-you-trust.md

```diff
@@ -6,55 +6,49 @@ keywords: ucp, dtr, security, trust
 
 >{% include enterprise_label_shortform.md %}
 
-With Docker Universal Control Plane you can enforce applications to only use
+With Docker Universal Control Plane (UCP), you can enforce applications to only use
 Docker images signed by UCP users you trust. When a user tries to deploy an
 application to the cluster, UCP checks if the application uses a Docker image
-that is not trusted, and won’t continue with the deployment if that’s the case.
+that is not trusted. If the image is not trusted, UCP will not continue with the deployment.
```

Comment:
Suggest editing the sentence to read: "Each time a user attempts to deploy an application to the cluster, UCP checks whether the application is using a trusted Docker image (and will halt the deployment if that is not the case)."